### PR TITLE
[tests-only][full-ci] refactor scenario example table

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
This PR refactors the scenario example tables in feature files.

## Related Issue
- https://github.com/owncloud/ocis/issues/5337

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/coreApiShareCreateSpecialToShares2/createShareReceivedInMultipleWays.feature [hunk 1]
```diff
@@ -214,20 +214,16 @@ Feature: share resources where the sharee receives the share in multiple ways
...
-    And user "Brian" accepts share "<path>" offered by user "Carol" using the sharing API
-    And user "Alice" accepts share "<path>" offered by user "Carol" using the sharing API
+    And user "Brian" accepts share "/child1" offered by user "Carol" using the sharing API
+    And user "Alice" accepts share "/child1" offered by user "Carol" using the sharing API
     Then the HTTP status code of responses on all endpoints should be "200"
     And the OCS status code of responses on all endpoints should be "100"
     And user "Brian" should be able to create folder "/Shares/parent/fo1"
     And user "Brian" should be able to create folder "/Shares/parent/child1/fo2"
     And user "Alice" should not be able to create folder "/Shares/child1/fo3"
-    @issue-2440
```
Comment (SwikritiT) [inline tests/acceptance/features/coreApiShareCreateSpecialToShares2/createShareReceivedInMultipleWays.feature:224]: please shift this issue to the top of the scenario

**Review by SwikritiT — COMMENTED** [review]
others lgtm

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Review by amrita-shrestha — APPROVED** [review]
lgtm

File: tests/acceptance/features/coreApiShareCreateSpecialToShares2/createShareReceivedInMultipleWays.feature [hunk 2]
```diff
@@ -193,7 +189,7 @@ Feature: share resources where the sharee receives the share in multiple ways
       | 2               | 200             |
 
 
-  Scenario Outline: sharing parent folder to user with all permissions and its child folder to group with read permission then check create operation
+  Scenario: sharing parent folder to user with all permissions and its child folder to group with read permission then check create operation
```
Comment (grgprarup) [inline tests/acceptance/features/coreApiShareCreateSpecialToShares2/createShareReceivedInMultipleWays.feature:192]:
```suggestion
  @issue-2440
  Scenario: sharing parent folder to user with all permissions and its child folder to group with read permission then check create operation
```

**Review by grgprarup — APPROVED** [review]
others. lgtm

**Review by saw-jan — COMMENTED** [review]
:+1:

**Review by saw-jan — APPROVED** [review]

**Pull request merged**